tests/drivers/ble_controller_clock/prj.conf

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
Should this test ble located under tests/subsys/bluetooth/controller?

Reply:
Sounds good to me

Reply:
Done